spec/sales_engine_spec.rb

```diff
@@ -12,4 +12,31 @@
 
     expect(se).to be_an_instance_of(SalesEngine)
   end
+
+  xit 'can create an items repository' do
+    se = SalesEngine.new({
+      :items     => "./data/items.csv",
+      :merchants => "./data/merchants.csv",
+    })
```

Comment:
add as a before :each for 
se = SalesEngine.new({      :items     =&gt; "./data/items.csv",      :merchants =&gt; "./data/merchants.csv",    })